11/Dockerfile

```diff
@@ -0,0 +1,44 @@
+# Base image for Gradle/Java/Springboot apps
+ARG version=11
+FROM gcr.io/distroless/java:${version}
+
+ENV APP_USER hmcts
+
+ONBUILD ARG JAVA_OPTS
+ONBUILD ENV JAVA_OPTS=$JAVA_OPTS
+
+ONBUILD ARG APP_INSIGHTS_AGENT_VERSION
+ONBUILD ENV APP_INSIGHTS_AGENT_VERSION=${APP_INSIGHTS_AGENT_VERSION:-2.3.1}
+
+ONBUILD ARG JAVA_AGENT_OPTIONS
+ONBUILD ENV JAVA_AGENT_OPTIONS=${JAVA_AGENT_OPTIONS:--javaagent:/opt/app/applicationinsights-agent-${APP_INSIGHTS_AGENT_VERSION}.jar}
+
+COPY group /etc/group
+COPY passwd /etc/passwd
+
+COPY --chown=hmcts:hmcts app/ /opt
+
+WORKDIR /opt/app
+USER $APP_USER
+
+COPY applicationinsights-agent-2.3.1.jar /opt/app
```

Comment:
Not sure if this is the best approach, but there's no wget or curl installed, and not sure if we would want to add it for this

Also app insights agent is broken on java 11, looking at fixing it.
see https://github.com/Microsoft/ApplicationInsights-Java/issues/897

Reply:
would a multi-stage build help here? (i.e. use something with cURL in it to download the binary)

Reply:
hmm yeah but i think that would need to be in the consumer image not this one